docs/transformation-modules/api/c-api.md

```diff
@@ -0,0 +1,144 @@
+---
+id: c-api
+title: Transformation modules C API
+sidebar_label: C API
+---
+
+This complements API documentation for `mg_procedure.h` which contains declarations of all functions that can be used to implement a transformation. The source file can be found in the Memgraph installation directory, under `include/memgraph`. On the standard Debian installation, this will be under `/usr/include/memgraph`. 
+
+:::caution
+**NOTE:** This part of the documentation is still under development. An updated version will soon be available.
+:::
+
+## Classes
+
+|                | Name           |
+| -------------- | -------------- |
+| struct | **mgp_label** <br/>Label of a vertex.  |
+| struct | **mgp_edge_type** <br/>Type of an edge.  |
+| struct | **mgp_property** <br/>Reference to a named property value.  |
+| struct | **mgp_vertex_id** <br/>ID of a vertex; valid during a single query execution.  |
+| struct | **mgp_edge_id** <br/>ID of an edge; valid during a single query execution.  |
+
+## Types
+
+|                | Name           |
+| -------------- | -------------- |
+| typedef void (*mgp_trans_cb)(const struct mgp_messages *, const struct mgp_graph *, struct mgp_result *, struct mgp_memory *); |
+  **[mgp_trans_cb](#typedef-mgp_trans_cb)** <br/>Entry-point for a transformation with a fixed result type |
```

Comment:
Escape all the asterix as you said.
Also, these 2 lines need to be in a single line so this is in a single row in a table. Right now it's rendered into 2 rows.